tests/python/gaia-ui-tests/gaiatest/visual_components/switch.py

```diff
@@ -0,0 +1,18 @@
+from marionette_driver import Wait
+from gaiatest.visual_components.visual_component import VisualComponent
+
+
+class Switch(VisualComponent):
```

Comment:
Would this support both switch and gaia-switch?  I think gaia-switch uses slightly different method to check its state.  Not sure whether youre is_checked() would work in case of gaia-switch